lib/src/create_report.dart

```diff
@@ -542,9 +536,14 @@ class _Issue {
 
   String markdown({@required String basePath}) {
```

Comment:
I agree with @jonasfj that this could be formatted better for a CLI tool, but I think this can came in a subsequent PR (or even a later release).

Reply:
Yeah, we don't need a CLI tool now. I was mostly suggesting we make it easier to review the output.